py/torch_tensorrt/dynamo/partitioning/common.py

```diff
@@ -16,6 +16,43 @@
 }
 
 
+def run_shape_analysis(
+    parent_module: torch.fx.GraphModule, inputs: Sequence[Input]
+) -> Tuple[Dict[Any, Sequence[Any]], Dict[Any, Sequence[Any]]]:
```

Comment:
I think this is okay to leave for now - in the future we may way to try using Torch's `ShapeProp` utility to handle this for us